connector/subgraph/src/DisputableVoting.ts

```diff
@@ -0,0 +1,213 @@
+import { STATUS_SETTLED, STATUS_SETTLED_NUM } from './statuses'
+
+/* eslint-disable @typescript-eslint/no-use-before-define */
+
+export function handleNewSetting(event: NewSettingEvent): void {
+  const votingApp = VotingContract.bind(event.address)
+  const settingData = votingApp.getSetting(event.params.settingId)
+
+  const currentSettingId = getVotingConfigEntityId(
+    event.address,
+    event.params.settingId
+  )
+  const votingConfig = getVotingConfigEntity(
+    event.address,
+    event.params.settingId
+  )
+
+  const daoAddress = votingApp.kernel()
+  const honeyPotConfig = loadOrCreateConfig(daoAddress)
+
+  votingConfig.settingId = event.params.settingId
+  votingConfig.voteTime = settingData.value0
+  votingConfig.supportRequiredPct = settingData.value1
+  votingConfig.minimumAcceptanceQuorumPct = settingData.value2
+  votingConfig.delegatedVotingPeriod = settingData.value3
+  votingConfig.quietEndingPeriod = settingData.value4
+  votingConfig.quietEndingExtension = settingData.value5
+  votingConfig.executionDelay = settingData.value6
+  votingConfig.createdAt = event.block.timestamp
+  votingConfig.save()
+
+  honeyPotConfig.voting = currentSettingId
+  honeyPotConfig.save()
+}
+
+export function handleStartVote(event: StartVoteEvent): void {
+  const votingApp = VotingContract.bind(event.address)
+  const proposal = getProposalEntity(event.address, event.params.voteId)
+
+  const voteData = votingApp.getVote(event.params.voteId)
+
+  proposal.type = PROPOSAL_TYPE_DECISION
+  proposal.typeInt = PROPOSAL_TYPE_DECISION_NUM
+  proposal.creator = event.params.creator
+  proposal.context = event.params.context.toString()
+  proposal.yeas = voteData.value0
+  proposal.nays = voteData.value1
+  proposal.totalPower = voteData.value2
+  proposal.startDate = voteData.value3
+  proposal.snapshotBlock = voteData.value4
+  proposal.status = castVoteStatus(voteData.value5)
+  proposal.statusInt = castVoteStatusNum(voteData.value5)
+  const settingsId = getVotingConfigEntityId(event.address, voteData.value6)
+  proposal.setting = settingsId
+  proposal.actionId = voteData.value7
+  proposal.challengeId = BigInt.fromI32(0)
+  proposal.challenger = Address.fromString(
+    '0x0000000000000000000000000000000000000000'
+  )
+  proposal.challengeEndDate = BigInt.fromI32(0)
+  proposal.pausedAt = voteData.value8
+  proposal.pauseDuration = voteData.value9
+  proposal.quietEndingExtensionDuration = voteData.value10
+  proposal.quietEndingSnapshotSupport = castVoterState(voteData.value11)
+  proposal.script = event.params.executionScript
+  proposal.settledAt = BigInt.fromI32(0)
+  proposal.disputedAt = BigInt.fromI32(0)
+  proposal.executedAt = BigInt.fromI32(0)
+  proposal.createdAt = BigInt.fromI32(0)
+  proposal.isAccepted = isAccepted(
+    voteData.value0, // yeays (using this instead proposa.yeays because we have the attribute as not required on the proposal entity)
```

Comment:
ah no indeed this is the exact reason why i added that comment, if you try to do this ```proposal.isAccepted = isAccepted(
    proposal.yeas, 
    voteData.nays, 
    voteData.totalPower,
    settingsId,
    votingApp.PCT_BASE()
  )```

yo will get this error when generating the code ``` ERROR AS200: Conversion from type '~lib/@graphprotocol/graph-ts/index/BigInt | null' to '~lib/@graphprotocol/graph-ts/index/BigInt' requires an explicit cast.

     proposal.yeas, // yeays
     ~~~~~~~~~~~~~
 in src/DisputableVoting.ts(102,4)
```

and as far as i understood is because we don't have the yeays, nays and any other attribute as required on the schema